Rulebook 3.0 (#113)
International-Data-Spaces-Association/IDSA-Rulebook: 004_Layers.md

## Layers of a Data Space

Data spaces can be structured into three primary layers, each serving distinct functions:

### Technical Layer
Encompasses the architecture and protocols (e.g., Dataspace Protocol (DSP), Decentralized Claims Protocol (DCP)) that facilitate trusted and interoperable data sharing. Software agents, representing the participants of the data space, form a decentralized mesh of autonomous nodes.

### Economic Layer
Manages the services, interactions, and workflows that enable value generation. Alternative terms for this layer are also *Business* or *Operational Layer*.

### Legislative Layer

> Enforces rights, obligations, and regulatory compliance across participants. Alternative terms for this layer are also *Regulatory* or *Governance Layer*.

Reviewer: Governance aspects, and especially the governance principles and rules on how the data space should operate, are also included in the Economic Layer. This relates to the various roles of the DSGA.

```suggestion
Enforces rights, obligations, and regulatory compliance across participants. Alternative term for this layer is *Regulatory Layer*.
```

Author: then we need to discuss the wording here. I want to distinguish between legal governance and business process. Business Process is in the Economic Layer (and might include business governance processes), but the Regulatory Layer is special as it enforces legal governance, which is mostly mandatory and could have severe consequences if not followed. We need to find a way to distinguish the more or less voluntary governance of the business from the mandatory legal governance.

## Participation and Representation

Participants in a data space are defined by their ability to automate the negotiation and execution of data sharing agreements via technical protocols. This has several implications:

- **Organisations**, not individuals, are considered technical participants. These organisations are represented by **software agents** capable of executing standardised data space protocols.

Note: For technical participation, "organisation" includes incorporated entities, not-for-profit bodies, public sector organisations, and registered sole traders or self-employed individuals acting as legal entities. Individuals may participate only when acting as a legal entity and if they are able to operate all required software agents directly or via an organisation that represents them (for example, a service provider).

> - **Natural persons** interact with data spaces indirectly through applications or services operated by organisations. Their participation is strictly limited to the Economic and Legislative Layer. Software agents can represent a natural person but not impersonate it.

Reviewer: ```suggestion
- **Natural persons** shall interact with data spaces only indirectly through applications or services operated by organisations, and their participation shall be limited to the Economic and Legislative Layer. The services may include e.g. personal data intermediation (including consent management) or data wallet provision. Software agents may act on behalf of a natural person, but shall not impersonate or assume the identity of that person in the technical layer.
```

> - As data spaces are fully decentralized and participants are responsible for their own **decentralized identity (DID)** there are **no identity providers** in a data space. Participants provide proof of their identity to others through the use of claims (e.g. expressed through verifiable credentials) and not through a common identity provider. Access to resources is managed through authentication tokens issued directly by the participants sharing those resources.

Reviewer 2: There's a comma missing:

> As data spaces are fully decentralized and participants are responsible for their own decentralized identities (DIDs), there are no identity providers in a data space.

## External Actors

Entities that provide static resources (such as ontologies, schemas, or public credentials) may support the data space but are not considered participants unless they actively engage via governed interfaces implementing the standardised data space protocols. For example:

- A web service that hosts a data sharing ontology is not a participant but serves as an **external reference**.

- A dataspace trust framework provider usually acts as an external reference. However, it may act as a **participant** if it delivers services subject to data space governance policies provided through an implementation of standardised data space protocols.

> - **Trust anchors**, **regulators** and similar legal entities may influence data transactions but do not participate directly unless they act through technical interfaces governed by data space rules. Most commonly they provide external services which are referenced in the data space (e.g. common data lookup services, registries, etc...)

Reviewer: ```suggestion
- **Trust anchors**, **regulators** and similar legal entities may influence data transactions but do not participate directly unless they act through technical interfaces governed by data space rules. Most commonly they provide external services which are referenced in the data space (e.g. common data lookup services, registries).
```

This model guarantees the adherance to the governance model and thus supports trust while preserving digital sovereignty of each participant and the integrity of technical interactions. Flexibility is created through processes in the economic layer which supports extensive customisation. The DSGA must not mandate central or federated services that restrict participant autonomy or agency without explicit justification; any such choice must be documented and accompanied by mitigation measures to preserve participant control.

Participation requires **governance commitment** and **technical integration**.

## Implications for the Rulebook

The Rulebook reflects these principles clearly:

> - The only **technical role** is the **participant**, which may act as data provider, consumer, or both.

Reviewer: ```suggestion
- The only **technical role** is the **participant**, which may act as data provider, data consumer, or both.
```
